docusaurus.config.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
         {
           position: "left",
           label: "Examples",
-          href: "https://github.com/neon-bindings/examples#table-of-contents",
+          href: "https://github.com/neon-bindings/examples/tree/legacy#table-of-contents",
```

Comment:
This causes prettier to fail:
https://travis-ci.org/github/neon-bindings/website/builds/769586180#L298. Need to break to new line